data/curated/chess_game_dataset/cleaning.R

```diff
@@ -0,0 +1,10 @@
+# Source: Clean data provided by Kaggle Mitchell J.
+# https://www.kaggle.com/datasets/datasnaek/chess/data
+
+library(tidyverse)
+
+chess <- readr::read_csv("data/curated/chess_game_dataset/chess_games.csv") %>%
```

Comment:
```suggestion
# Data saved from Kaggle as "chess_game_dataset/chess_games.csv"

chess <- readr::read_csv("chess_game_dataset/chess_games.csv") %>%
```

The "curated" path won't exist once this is merged, so I try to include the download whenever possible. I know Kaggle makes it hard, so in that case it's ok to do whatever you have to do for the actual cleaning, and then simplify the script for the final post.